video/out/wayland_common.c

```diff
@@ -1410,28 +1410,23 @@ static void check_dnd_fd(struct vo_wayland_state *wl)
         return;
 
     if (fdp.revents & POLLIN) {
-        ptrdiff_t offset = 0;
         size_t data_read = 0;
         const size_t chunk_size = 1;
```

Comment:
reading it byte-by-byte, really?

Reply:
This is a change for another PR to not mix it with unrelated changes. Maybe there was some reason to read it byte-by-byte. Don't know.

Reply:
probably not, but can be fixed in another PR.